hs_core/management/utils.py

```diff
@@ -215,6 +218,52 @@ def check_relations(resource):
                       .format(resource.short_id, r.type, target))
 
 
+def fix_resourcefile_duplicates(dry_run=False, logger=None, get_model=False):
+    """Remove duplicate ResourceFiles
+
+    Args:
+        dry_run (bool, optional): True means don't save any changes. Defaults to False.
+    """
+    if not logger:
+        logger = logging.getLogger(__name__)
+    # first we remove files with content_type other than the content type for CompositeResource
+    if get_model:
+        CompositeResourceModel = apps.get_model('hs_composite_resource', 'CompositeResource')
+        ResourceFileModel = apps.get_model('hs_core', 'ResourceFile')
+    else:
+        from hs_core.models import ResourceFile as ResourceFileModel
+        from hs_composite_resource.models import CompositeResource as CompositeResourceModel
+    desired_content_type = ContentType.objects.get_for_model(CompositeResourceModel)
```

Comment:
Use apps.get_model() for ContentTYpe:
`ContentType = apps.get_model('contenttypes', 'ContentType')`

Reply:
did that in https://github.com/hydroshare/hydroshare/pull/5191/commits/d61680e7597996d4243fdb209a8e11a42ae60cdc thanks